src/js/program/program-controller.js

```diff
@@ -45,6 +45,7 @@ class ProgramController extends Eventable {
         const item = model.get('playlist')[index];
 
         model.setActiveItem(index);
+        model.set('itemReady', false);
```

Comment:
This should be set before `setActiveItem`. To prevent this from trigging a change event, set it directly on the `model.attributes`:
  
```js
model.attributes.itemReady = false;
model.setActiveItem(index);
```